cmd/soroban-cli/src/config/locator.rs

```diff
@@ -468,14 +468,22 @@ impl KeyType {
 }
 
 pub fn global_config_path() -> Result<PathBuf, Error> {
-    Ok(if let Ok(config_home) = std::env::var("XDG_CONFIG_HOME") {
+    let config_dir = if let Ok(config_home) = std::env::var("XDG_CONFIG_HOME") {
         PathBuf::from_str(&config_home).map_err(|_| Error::XdgConfigHome(config_home))?
     } else {
         dirs::home_dir()
             .ok_or(Error::HomeDirNotFound)?
             .join(".config")
+    };
+
+    let soroban_dir = config_dir.join("soroban");
+    let stellar_dir = config_dir.join("stellar");
+
+    if soroban_dir.exists() {
```

Comment:
If both exists, IMO we should either use `stellar_dir` first or return an error

Reply:
Good catch. Also, we should log a warning message in case both directories exist, just like the local config is doing right now.